tests/try_into.rs

```diff
@@ -0,0 +1,79 @@
+#![cfg(feature = "nightly")]
+#![feature(try_from)]
+#![allow(dead_code)]
+
+#[macro_use]
+extern crate derive_more;
+
+use std::convert::TryInto;
+
+#[derive(Clone, Copy, TryInto)]
+enum MixedInts {
+    SmallInt(i32),
+    NamedBigInt { int: i64 },
+    TwoSmallInts(i32, i32),
+    NamedBigInts { x: i64, y: i64 },
+    Unsigned(u32),
+    NamedUnsigned { x: u32 },
+    Unit,
+}
+
+#[test]
+fn test_try_into() {
+    let i = MixedInts::SmallInt(42);
+    assert_eq!(Ok(42i32), i.try_into());
+    assert_eq!(TryInto::<i64>::try_into(i), Err("Only NamedBigInt can be converted to i64"));
```

Comment:
I think these errors can probably be done using `i64::try_from()`. Like you did in the example. If so can you use that and also remove the `use std::convert::TryInto;` line.